src/com/clover/remote/client/transport/websocket/CloverWebSocketClient.ts

```diff
@@ -30,6 +30,10 @@ export class CloverWebSocketClient implements WebSocketListener {
         return (this.socket) ? this.socket.getReadyState() : null;
     }
 
+    public getBufferedAmount(): number {
+        return (this.socket ? this.socket.getBufferedAmount(): 0);
```

Comment:
Will getBufferedAmount() be present in all implementations?  Any chance of an error because of missing functionality?

Reply:
No, it may not be.  But I figure in those cases (Node) the code in this.socket (CloverWebSocketInterface) can check for the existence of bufferedAmount and return 0 if it doesn't exist.  My concern with it is the following from Mozilla's developer docs:

"The number of bytes of data that have been queued using calls to send() but not yet transmitted to the network. This value resets to zero once all queued data has been sent. This value does not reset to zero when the connection is closed; if you keep calling send(), this will continue to climb. Read only"